Makefile

```diff
@@ -2,6 +2,7 @@ PYTHON = python
 .PHONY: docs build perf
 
 OS := $(shell uname)
+HAS_GIT := $(shell command -v git 2> /dev/null)
```

Comment:
Instead of doing this magic :) let's add precommit to setup.py in the dev dependencies. I assume that if you are developing thumbor you should abide by these rules, right?

Reply:
And there's no need to install for every user anyway, right?

Reply:
There is a error when running `pre-commit install`, because there is no .git folder or git command in CI, for that reason I did this magic =)

Reply:
is there another better way to install hooks?

Reply:
I was actually looking into a way to check whether a commit was run by pre-commit. There's a pre-sign pre-commit plugin that embeds in the commit a signature that proves that the user has actually run pre-commit. We could check this in PRs. I'm not sure we can enforce people running pre-commit anyway.  WDYT?

Reply:
Instead of run `pre-commit install` in `make setup` we can show a message asking user to run it, what do you think?!

Reply:
I like it :)